build/BUILD.bazel

```diff
@@ -133,6 +140,25 @@ kong_directory_genrule(
         LN ${BUILD_DESTDIR}/openresty/bin/resty ${BUILD_DESTDIR}/bin/resty
         chmod -R "+rw" ${BUILD_DESTDIR}/openresty
 
+        CURL=${WORKSPACE_PATH}/$(echo '$(locations @curl//:curl)' | awk '{print $1}')
```

Comment:
i would suggest to use a different prefix for those tools, as it will be confusing for customer (or future us) to realize kong depends on curl.
this can also helps
1) nfpm exclude them from packaing
2) speed up local development and skip building curl

Reply:
If you read through the ticket, the idea is to have these tools be available in both the packages and containers of Kong so that customer-facing Kongers (and customers) can have them to troubleshoot issues wherever Kong is installed, without the need to install additional tools.

Equivocally, that makes curl, and whatever else we add for this purpose, a dependency of the gateway.

Since this curl binary isn't meant to replace the system's, I think it makes sense to have it in a prefix underneath Kong's /usr/local/kong. I was also planning on leaving it off of $PATH by default, to cut down on the possibility of someone thinking this is the system's curl, and not one specific to kong.

Reply:
Correct me if I'm wrong, but I don't think we have a way within our bazel build to discern if we are doing local development or not. I would think we'd need something like a `select` compatible flag. Even targeting `@platforms//os:macos` isn't enough, because some developers use linux.